Session 2/Concept 2/index.html

```diff
@@ -0,0 +1,38 @@
+<html>
+<head>
+    <title>Chef's Table</title>
+</head>
+<body>
+    <div class="header">
```

Comment:
`class` and `id` should not be used until CSS is taught.
Make this change throughout the file.